mem-08-m.xml

```diff
@@ -4778,7 +4778,9 @@ Due to the influence of {mIQ:v}, this verb is used differently in the context of
       <column name="notes_ru"></column>
       <column name="notes_zh_HK"></column>
       <column name="notes_pt"></column>
-      <column name="notes_fi">Tällä verbillä on kolme merkitystä. Se on joko intransitiivinen "palaa" (esim. {meQtaHbogh qachDaq Suv qoH neH.:sen}[2]), jolla on normaalisti kausatiivi "polttaa, sytyttää tuleen" {meQmoH:v:nolink}[3]. Ruoan yhteydessä sillä on myös transitiivinen merkitys "polttaa, kypsentää" (esim. {to'waQ meQ vutwI'.:sen@@to'waQ:n, meQ:v:2, vutwI':n}) ja adjektiivinen merkitys "olla poltettu, kypsennetty" (esim. {Ha'DIbaHmey meQ Sop 'e' tIv tera'nganpu'.:sen}).</column>
+      <column name="notes_fi">"Polttaa" on {meQmoH:v@@meQ:v:1, -moH:v}[6].
```

Comment:
The sources were renumbered after the split. I think you have to renumber [6] to [4] in the Finnish notes.